### PR TITLE
add reproducible_bug_buckets data to testing_summary.json

### LINKED ISSUE #345: Add bug_bucket reproducibility data to testing_summary.json
Data on bug reproducibility should be added to testing_summary.json, to facilitate automatic data collection and post-processing after RESTler runs.

A PR for this is coming up shortly.

### PR DESCRIPTION
**Pull request by PatGod**
closes #345 

Testing: manually

Note: because new data is added to testing_summary.json without modifying the format for previously existing data, these new edits/additions should be backward compatible to existing post-processing tools for testing_summary.json

### CODE REVIEW

File: src/driver/Types.fs [hunk 1]
```diff
@@ -167,7 +167,10 @@ module Engine =
     //        "main_driver": 66
     //    },
     //    "bug_buckets": {
-    //        "main_driver_500": 0
+    //        "main_driver_500": 1
+    //    },
+    //    "reproducible_bug_buckets": {}
```
Comment (marina-p) [inline src/driver/Types.fs:172]: >} [](http://example.com/codeflow?start=39&length=1)

remove extra closing brace

**Review by marina-p — DISMISSED** [review]
:shipit:

File: src/driver/Types.fs (hunk 1 above)
Comment (PatGod) [reply src/driver/Types.fs]: Done!

**Comment by PatGod** [comment]
@marina-p In Types.fs, I saw this comment: "These statistics will be reported in Microsoft telemetry"

Can you please confirm that adding this new data won't break the telemetry? (I didn't test the latter.)

**Comment by marina-p** [comment]
Yes, it just won't be reported in telemetry.   Only the data in 'bug_buckets' will be reported.

(This is implemented in 'getDataFromTestingSummary' in Program.fs)

---
In reply to: [916389502](https://github.com/microsoft/restler-fuzzer/pull/346#issuecomment-916389502) [](http://example.com/codeflow?ancestors=916389502,916389502)

**Comment by PatGod** [comment]
@marina-p Thanks much for confirming. (It is fine not to upload this new data, one can always revisit this later if needed....)

**Pull request merged**